asapdiscovery-dataviz/asapdiscovery/dataviz/gif_viz.py

```diff
@@ -116,8 +121,9 @@ def __init__(
         self.start = start
         self.stop = stop
         self.interval = interval
-
+        self.bool_static_view_only = True
```

Comment:
This is set but not re-used? I would re use in function below or remove

Reply:
right, I changed it to using `self` throughout.